apps/ecommerce/lambda/src/app.ts

```diff
@@ -23,17 +24,17 @@ app.use(Middleware.setSentryContext);
 app.use(Sentry.Handlers.requestHandler() as express.RequestHandler);
 
 app.use(apiRouteConstraint, cors(corsConfig)); // enable CORS on /api/* routes
+
+// TODO: mount headers before verify
 app.use(apiRouteConstraint, Middleware.loadAppConfig); // load app config on /api/* routes
 app.use(apiRouteConstraint, Middleware.verifiySignedRequests); // verify signed requests on /api/* routes
```

Comment:
how will verification of signed requests be handled as part of this PR? (Just going off the knowledge of having to comment it out yesterday to get it